examples/js/trx/create-address.js

```diff
@@ -0,0 +1,24 @@
+const BitGoJS = require('bitgo');
+const Promise = require('bluebird');
+
+// TODO: change to 'production' for mainnet
+const env = 'test';
+const bitgo = new BitGoJS.BitGo({ env });
+
+// TODO: change to 'trx' for mainnet
+const coin = 'ttrx';
+
+// TODO: set your wallet id
+const walletId = '';
+
+// TODO: set your access token here
+// You can get this from User Settings > Developer Options > Add Access Token
+const accessToken = '';
+
+Promise.coroutine(function *() {
```

Comment:
Using Coroutines / blue bird is not required, we can use simple async / awaits.

Reply:
ok, updated.